hack/ci/e2e.sh

```diff
@@ -144,7 +146,7 @@ data:
     .:53 {
         errors
         health
-        kubernetes cluster.local in-addr.arpa ip6.arpa {
+        kubernetes cluster.local internal in-addr.arpa ip6.arpa {
```

Comment:
can we comment concretely on what the diff is vs the default config?

Reply:
done